docs/topics/mta-web-config-jira-credentials.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-config-jira-credentials_{context}"]
+= Configuring Jira credentials
+
+You need to configure a Jira credential for establishing a connection to Jira and creating Jira issues from within the {ProductName} ({ProductShortName}) {WebName} for tracking application migrations.
+
+.Procedure
+
+. In *Administration* view, click *Credentials*.
++
+The *Credentials* page opens.
+. Click *Create new*.
+. Enter the following information:
+    * Name
+    * Description (optional)
+. In the *Type* list, select *Basic Auth (Jira)* or *Token Bearer (Jira)*:
++
+* *Basic Auth (Jira)* - used for Jira Cloud
++
+If you selected *Basic Auth (Jira)*, proceed as follows:
+
+.. In the *Email* field, enter your email.
+.. In the *Token* field, enter your token generated on the Jira site.
++
+[NOTE]
+====
+To obtain a Jira token, you need to log in to the Jira site and click *Profile* > *Personal Access Tokens* > *Create*.
+====
++
+.. Click *Save*.
++
+The new credential appears on the *Credentials* page.
+
+* *Token Bearer (Jira)* - used with a private Jira server or datacenter
++
+If you selected *Token Bearer (Jira)*, proceed as follows:
+
+.. In the *Token* field, enter a token (??).
+.. In the *Key* field, enter the key (??)
```

Comment:
Those are token or key generated on Jira side. 
In case of Cloud Jira connection - combination of Username (email actually) and Token should be used.  - this is called basic authentication. 
In case of Datacenter/Server account - either basic auth or Bearer token (marked as key) should be used. In any case, as I mentioned, both are generated on Jira side and the process is well documented.

Reply:
@ibragins Thank you!
I am not sure I understand correctly:
If the user selects the *Token bearer* type (for a private Jira server or datacenter), what does he enter in the *Token* field and what in the *Key* field?